### ISSUE TITLE
TensorFlow Federated: Machine Learning on Decentralized Data

### ISSUE THREAD

**Issue by StuartIanNaylor** [TensorSpeech/TensorFlowASR#228]
Purely a question but just wondering if you guys had any thoughts on localised federated training for TensorflowASR?

https://www.tensorflow.org/federated

The new Pixel6 phones with its TPU name dropped federated learning and always local federated training data, which seems a panacea to the problems of privacy?
I am completely unsure of limitations and implementation but very interested as very aware of the accuracy increases of usage data being added to a general model.

Just wondered if this is on the TensorflowASR radar?

**Comment by usimarit** [comment]
@StuartIanNaylor It's not on our radar yet but might be in the future. For now we are focusing on models and the code base so that they are scalable and less bug 😄
